### ISSUE TITLE
LocalDate.now fails if no tzdb in scala.js

### ISSUE THREAD

**Issue by pmeheut** [cquiroz/scala-java-time#335]
LocalDate.now calls Clock.systemDefaultZone. On my machine, it finds that the timezone is Europe/Paris and I end up with an exception:

```
Uncaught java.time.zone.ZoneRulesException: Unknown time-zone ID: Europe/Paris
    at $p_Ljava_time_zone_ZoneRulesProvider$__getProvider__T__Ljava_time_zone_ZoneRulesProvider (http://localhost:8080/assets/main.js:9479:13)
```

I can solve it by using tzdb but this is an overkill when all I want is the current year, month and day.

**Comment by cquiroz** [comment]
I see, I guess `LocalDate` needs the zone to find when is `now` right?

**Comment by pmeheut** [comment]
Yes but it should not need a database. I solved it by implementing different version for jvm and js.
For jvm:
```
def safeNow(): LocalDate = LocalDate.now()
```

for js:
```
 def safeNow(): LocalDate = {
  val d = new scala.scalajs.js.Date()
  LocalDate.of(d.getFullYear().toInt, d.getMonth().toInt+1, d.getDate().toInt)
```

**Comment by cquiroz** [comment]
Right, I don't know the details but I guess calling `js.Date` would use the time zone of the environment. I imagine you need to know your timeZone in order to find the local time.

Anyway your solution should
Work and avoid the need of an extra tz db
